pyrokinetics/gk_code/GKOutputReader.py

```diff
@@ -278,4 +322,90 @@ def _set_growth_rate_tolerance(data: xr.Dataset, time_range: float = 0.8):
         return data
 
 
+def eigenvalues_units(convention: Convention):
+    """
+    Generates dictionary of flux output units
+    Parameters
```

Comment:
You'll want a blank line here to render this properly in the docs:

```suggestion

    Parameters
```

And in the routines below.


Turns out we don't actually build docs for the complete package, so I'll add those in a separate PR